src/app/pages/property-details/confirmation-modal/confirmation-modal.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, OnInit, Output, EventEmitter } from '@angular/core';
+import { ToastrService } from 'ngx-toastr';
+import { ActivatedRoute, Router } from '@angular/router';
+import { PropertyDetailService } from 'src/app/services/property-detail/property-detail.service';
+
+@Component({
+  selector: 'app-confirmation-modal',
+  templateUrl: './confirmation-modal.component.html',
+  styleUrls: ['./confirmation-modal.component.scss']
+})
+export class ConfirmationModalComponent implements OnInit {
+  close_Modal: boolean;
```

Comment:
Hey @MBenedicte, I believe we can do better at this variable name.
1. we can use camelCase
2.since its a boolean, we can name it as `isModalOpen` or `modalOpen` or any other in that line.

Reply:
worked on that @CryceTruly